client/src/app/(modules)/practices/(main)/(details)/[id]/page.tsx

```diff
@@ -1,6 +1,6 @@
 import { notFound } from 'next/navigation';
 
-import { ExternalLink } from 'lucide-react';
+import { ExternalLink, NotepadText, Tractor } from 'lucide-react';
```

Comment:
Use the custom icons instead (they are thinner): https://www.figma.com/file/tvQfQ21Gu8n4kK2xwjdkqp/ORCaSa---Design-System-v.2?type=design&node-id=224-1806&mode=design&t=KYBLbdSx4DEqjJnD-0